### ISSUE TITLE
PostGreSQL Connection Shows Empty Screen

### ISSUE THREAD

**Issue by EasternYou** [outerbase/studio#193]
Connection Profile written correctly, but when I attempt to connect I get a white screen like so:
![20241210_Outerbase_0 1 13_Blank_Screen](https://github.com/user-attachments/assets/6ca7f4e4-9cd1-4fe4-adae-ea5164ee4b1a)
Can you please investigate?
Thanks

**Comment by invisal** [comment]
Interesting. Is it local Postgres? If it is cloud, which cloud provider are you using (I will try to connect to them to check).

**Comment by EasternYou** [comment]
Bonterra is the provider

**Comment by EasternYou** [comment]
It’s via a Penelope database - https://intercom.help/ssgpenelope/en/articles/5189646-connecting-to-your-penelope-database

**Comment by EasternYou** [comment]
Appears to still be an issue for v0.15 of outerbase desktop

**Comment by invisal** [comment]
@EasternYou Is it possible for you provide me which demo access to your provider? I cannot register Penelope and I am pretty determined to make it work.

**Comment by EasternYou** [comment]
No it’s not sorry it’s for a restricted client

**Comment by invisal** [comment]
> No it’s not sorry it’s for a restricted client

In v0.1.15, there is "Connect with Debugger", and check the log. If there is something helpful information.

**Comment by EasternYou** [comment]
This is all the debugger shows in the Console when I attempt to connect.
Connection ID ab1568b9-4cc7-42c8-920a-a2d99646870b

**Comment by EasternYou** [comment]
Hi @invisal I just tried **test connection** on Outerbase Studio Desktop v0.1.21 and it connected successfully.
This means the UI isn't recognising the test connection and opening it after I try to connect.
It also means the fault isn't with the provider, it's with the application not recognising the Postgres connection.
Thanks,
EasternYou.

**Comment by invisal** [comment]
What I suspect is not enough permission to perform certain query. 

The test connection basically connect to provider and execute "SELECT 1 + 1".

But the GUI needs more than just ability to do query. It needs to fetch list of database, tables, columns, etc. Do you have success with connect your database with other GUI such as TablePlus, pgAdmin? I wonder if it is problem with just Outerbase Studio or problem with overral GUI.

**Comment by EasternYou** [comment]
Yes it works on pgAdmin, DBeaver, BeekeeperStudio, DBGate and other pg UIs. It works on TablePlus too. This is the only one where it fails.
Thanks,
EasternYou


Sent from Yahoo Mail for iPhone


On Wednesday, January 8, 2025, 1:38 pm, Visal .In ***@***.***> wrote:




What I suspect is not enough permission to perform certain query.

The test connection basically connect to provider and execute "SELECT 1 + 1".

But the GUI needs more than just ability to do query. It needs to fetch list of database, tables, columns, etc. Do you have success with connect your database with other GUI such as TablePlus, pgAdmin? I wonder if it is problem with just Outerbase Studio or problem with overral GUI.

**Comment by jamesysilber** [comment]
Hi @invisal this is still an issue - any chance it can be resolved? Thanks
